### DISCUSSION TITLE
Managed Milvus service

### DISCUSSION THREAD

**Discussion by mausch** [milvus-io/milvus#19088] [Ideas & Feature requests]
Hi, are there any plans to build a managed Milvus service?

Milvus has a lot of moving parts, especially in clustered mode. Many companies would gladly pay for a managed service rather than hosting and maintaining this internally.

I think Milvus is popular enough by now to make this worth investing into.

**Comment by yhmo** [comment]
Yes, we have built a Milvus service on cloud. We are inviting users to get started.
Here is the registration entry of the Milvus cloud: https://cloud.zilliz.com/login
